api-reference/openapi.json

```diff
@@ -2232,7 +2232,7 @@
           },
           "sleeping": {
```

Comment:
@athoscouto @avinassh do we plan to update the API too? Otherwise it’s just as confusing if we don’t.

Reply:
I have not made changes to the API to keep backward compatibility. 

I will send a follow up patch and update the API spec and replace `sleeping` with `archived`. On the API server, I will send both the fields. WDYT?

Reply:
Sending both fields sounds good to me. We can then update the docs to use archived and never speak of sleeping again 😂

Reply:
what is `sleeping`